### ISSUE TITLE
Cannot link more than one resource to a dataset using Dataset REST API

### ISSUE THREAD

**Issue by jakubklimek** [GetDKAN/dkan#1313]
## Description

When trying to create a dataset with more than one resource using the new Dataset REST API, I only manage to link one.
## Steps to Reproduce
- Create 2 resources (lets say with IDs 230 and 231)
- POST to /api/dataset/node with the X-CSRF-Token in Header and these in form data:

```
type:dataset
title:Another dataset
field_resources[und][0][target_id]:sdasd (231)
field_resources[und][1][target_id]:asddasasd (230)
```
-  I see a new dataset with one resource, the one from `field_resources[und][0][target_id]`.
## Acceptance Criteria

I would expect to see a dataset with 2 resources.

**Comment by dafeder** [comment]
Thanks @jakubklimek, to be honest we had only tested with a single resource so far! Will address this ASAP.

**Comment by jakubklimek** [comment]
Hi, it has been 3 months, any updates?

**Comment by dafeder** [comment]
@jakubklimek sorry for the lack of activity on this - we've had to reassess how to use services on DKAN as there are some issues with the structure of data it accepts (requiring incoming data to pass through the Form API) that we hadn't anticipated. This is being addressed right now in #2196
